src/v2/Components/Artwork/Details.tsx

```diff
@@ -1,10 +1,15 @@
-import { Flex, Link, Text } from "@artsy/palette"
+import {
+  Flex,
+  Link,
+  Text,
+  LinkProps,
+  useThemeConfig,
+  TextVariant,
+} from "@artsy/palette"
 import { Details_artwork } from "v2/__generated__/Details_artwork.graphql"
-import { SystemContextConsumer } from "v2/Artsy"
```

Comment:
Zero clue why this was being used. No children components needed the user. Maybe they did at one point?